packages/components/src/styles/paragraph/paragraph.stories.ts

```diff
@@ -41,22 +44,39 @@ export const Default = {
   }
 };
 
+/**
+ * Use the `sd-paragraph--size-sm` class to set a smaller font size.
+ */
+
+export const Size = {
```

Comment:
Can we add this?
"
Large is the default paragraph size
sd-paragraph--size-sm: Small can be used as an alternative"

Reply:
Updated